src/Type/Accessory/AccessoryArrayListType.php

```diff
@@ -247,6 +247,15 @@ public function shuffleArray(): Type
 		return $this;
 	}
 
+	public function sliceArray(Type $offsetType, Type $lengthType, TrinaryLogic $preserveKeys): Type
+	{
+		if ($offsetType instanceof ConstantIntegerType && $offsetType->getValue() === 0) {
+			return $this;
+		}
+
+		return $this;
```

Comment:
It returns this in the two situations, so the if check is useless, isn't it ?

Reply:
thx, yeah I messed this implementation somehow up with reverting parts and finally missing things.. should be good now :)